main.nf

```diff
@@ -302,16 +270,34 @@ if( params.include_proteins_from_vcf){
         .fromPath( params.vcf )
         .ifEmpty { exit 1, "params.vcf was empty - no input file supplied" }
         .into { input_vcf; input_vcf_neoepitope; input_vcf_neoepitope_II}
-} else {
 
+} else {
     input_vcf = Channel.empty()
     input_vcf_neoepitope = Channel.empty()
     input_vcf_neoepitope_II = Channel.empty()
 }
 
+// Stage config files
+ch_output_docs = file("$baseDir/docs/output.md", checkIfExists: true)
+
+// Has the run name been specified by the user?
+//  this has the bonus effect of catching both -name and --name
+custom_runName = params.name
+if (!(workflow.runName ==~ /[a-z]+_[a-z]+/)) {
+    custom_runName = workflow.runName
+}
+
+if (workflow.profile.contains('awsbatch')) {
+    // AWSBatch sanity checking
+    if (!params.awsqueue || !params.awsregion) exit 1, "Specify correct --awsqueue and --awsregion parameters on AWSBatch!"
+    // Check outdir paths to be S3 buckets if running on AWSBatch
+}
+
 // Header log info
 log.info nfcoreHeader()
 def summary = [:]
+if (workflow.revision) summary['Pipeline Release'] = workflow.revision
+summary['Run Name']         = custom_runName ?: workflow.runName
 summary['Pipeline Name']  = 'nf-core/mhcquant'
 summary['Pipeline Version'] = workflow.manifest.version
```

Comment:
Indents are off here in this section. Easier to visualise if you can try and line up the `=`

Reply:
https://github.com/Zethson/MHCquant/commit/51c5b888b894f1082ba1a08fdb0f80e608c49860